add 2020 recap post (#249)
nim-lang/website: jekyll/_posts/2020-12-28-nim-in-2020-a-short-recap.md

## Version 1.2

Nim version 1.2.0 was [released in April](https://nim-lang.org/blog/2020/04/03/version-120-released.html).
It contained around 600 new commits which have not already been backported to our
1.0.x versions and, among other things, introduced the **ARC memory management** and
several useful macros in the [sugar module](https://nim-lang.org/docs/sugar.html):
- `collect` (for creating list/set/table-comprehensions)
- `dup` (turning an in-place function into one that returns a result without modifying its input)
- `capture` (capturing local loop variables)

## Nim Conf 2020

<p align="center">
<img width="300px" style="background: #232733;" src="/assets/img/nim-conf-2020.svg">
</p>

Due to COVID-19 situation, we have decided that the first ever Nim conference
should be held online and be available for free for everybody.

It was held on June 20th via Youtube streams, containing **15 interesting talks**
where the authors showed wide variety of things they are developing using Nim.
Every talk also had a live chat where the authors and the audience interacted,
which proved to be very useful to get answers to some specific questions and
go into more depth about the topic.

> All talks can still be viewed on

Reviewer: ```suggestion
All the talks can still be viewed on
```

[the Youtube playlist](https://www.youtube.com/playlist?list=PLxLdEZg8DRwTIEzUpfaIcBqhsj09mLWHx)

> and we recommend you to take a look if you haven't watched the talks already.

Reviewer: ```suggestion
and we recommend that you take a look if you haven't watched them already.
```
